web/main/browser.ts

```diff
@@ -11,15 +11,28 @@ import {
 
 import { locale } from '../stores/locale.js'
 
+const actualizeThemeColor = (themeColorTag: Element | null): void => {
+  let backgroundColor = window
+    .getComputedStyle(document.body)
+    .getPropertyValue('background-color')
+
+  if (themeColorTag && backgroundColor) {
+    themeColorTag.setAttribute('content', backgroundColor)
+  }
+}
+
 let root = document.documentElement
+let themeColorTag = document.querySelector('meta[name="theme-color"]')
```

Comment:
Let’s use just `theme`. I prefer short variable names.

Reply:
Makes sense, I made it `themeTag` since we already have `theme` as an imported variable